politeiad/backendv2/tstorebe/plugins/pi/hooks.go

```diff
@@ -172,6 +173,10 @@ func (p *piPlugin) proposalDomainIsValid(domain string) bool {
 	return found
 }
 
+func (p *piPlugin) updateTitleIsValid(title string) bool {
```

Comment:
`proposalNameIsValid` and `updateTitleIsValid` do the same thing.  Remove one of them and rename the remaining one to be consistent with the plugin setting naming.